packages/store/src/__tests__/migration.unit.spec.ts

```diff
@@ -73,4 +73,17 @@ describe('migration', () => {
     expect(hasSurface).toBe(true);
     expect(hasShape).toBe(false);
   });
+
+  test('migrate to new page title', async () => {
+    const doc = await loadBinary('legacy-page-title');
+
+    assert.equal(
```

Comment:
This may not reflect the potential breaking change, hope for investigation @Flrande